Add PharaoHansen-2020-100 and Miller-1984-100

Kristina-Pianykh opened the pull request:
# Pull request checklist

- [x] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information
These are almost identical lists as Miller was used as the source list for PharaoHansen-2020-100 so I decided to open one PR for them. Both represent the basic vocabulary (Miller is based almost entirely on Swadesh-1955-100).

<details>
<summary>Concepticon check Miller-1984-100</summary>

```
concepticondata\conceptlists\Miller-1984-100.tsv
Check: unique_concepticon_gloss
OK
Check: unique_id
OK
Check: unique_number
OK
Check: matching_concepticon_gloss_and_id
OK
Check: valid_concepticon_gloss
OK
```

</details>

<details>
<summary>Concepticon check PharaoHansen-2020-100</summary>

```
concepticondata\conceptlists\PharaoHansen-2020-100.tsv
Check: unique_concepticon_gloss
OK
Check: unique_id
OK
Check: unique_number
OK
Check: matching_concepticon_gloss_and_id
OK
Check: valid_concepticon_gloss
OK
```

</details>

Kristina-Pianykh commented:
Could I ask you @AnnikaTjuka to moderate this PR if you're available?

AnnikaTjuka commented:
Sure, will do. @CarolinHu Do you have time to review this list? @LinguList as our Spanish expert, would you also please take a look?

CarolinHu commented on concepticondata/conceptlists.tsv:
```diff
@@ -330,3 +330,5 @@ Baroni-2011-200	Baroni, Marco and Lenci, Alessandro	2011		200	relations	English
 Luangthongkum-2019-341	Luangthongkum, Theraphan	2019		341	questionnaire, areal	English	Karenic varieties spoken in Thailand	https://evols.library.manoa.hawaii.edu/bitstream/10524/52441/D01Luangthongkum2019view.pdf	Luangthongkum2019		The given 341-item list was created in the reconstruction of the phonology and lexicon of Proto-Karen, based on the ten selected Karenic varieties spoken in Thailand. ...
 Pepper-2020-100	Pepper, Steve	2020		100	specific	English	Global		Pepper2020		The items in this list were used for a typological investigation of the morphology and semantics of binominal lexemes, i.e. of concepts typically expressed by nominal compounds, across 106 different languages.	393-394	
+PharaoHansen-2020-100	Pharao Hansen, Magnus	2020		100	basic, areal	Spanish	Southern Uto-Aztecan languages		Pharao2020		The given 100-item list was initially taken from [Miller-1984-100](:ref:Miller-1984-100) and was used in the preliminary analysis of the relationship between the six selected languages within the Southern Uto-Aztecan branch based on a wide range of empirical data.		
+Miller-1984-100	Miller, Wick R.	1984		100	basic, areal	Spanish	Uto-Aztecan languages		Miller1984	Miller1984	The current list represents a modified version of (Swadesh-1955-100)[:ref: Swadesh-1955-100] with twelve substitutions (*belly button*, *heavy*, *old*, *rope*, *salt*, *sing*, *sky*, *snake*, *snow*, *vomit*, *wind*, and *year* replace *all*, *full*, *I*, *round*, *say*, *swim*, *that*, *th ...
```
There's a space after :ref:

CarolinHu commented on concepticondata/conceptlists/Miller-1984-100.tsv:
```diff
@@ -0,0 +1,101 @@
+Miller-1984-100-10	10	bueno	good	1035	GOOD
+Miller-1984-100-11	11	pesado	heavy	1210	HEAVY
+Miller-1984-100-12	12	seco	dry	1398	DRY
+Miller-1984-100-13	13	frío	cold	2483	COLD (OF WEATHER)
```
Frío is usually mapped to COLD, is there a special reason you went for COLD (OF WEATHER) here?

Kristina-Pianykh replied:
I think I should change it to COLD as this list is claimed to bear a strong resemblance to Swadesh 1955.

CarolinHu commented on concepticondata/conceptlists/Miller-1984-100.tsv:
```diff
@@ -0,0 +1,101 @@
+Miller-1984-100-22	22	culebra, víbora	snake	730	SNAKE
+Miller-1984-100-23	23	árbol, palo	tree	906	TREE
+Miller-1984-100-24	24	corteza, cascara	bark	1204	BARK
+Miller-1984-100-25	25	hoja	leef	628	LEAF
```
Is the typo "leef" in the original document?

Kristina-Pianykh replied:
I don't have access to the original data on PharaoHansen-2020-100 (I took the concept list from the corresponding Lexibank repository) but I see @LinguList already checked it. In Miller, however, this typo should be fixed. Thanks for spotting it!

CarolinHu commented on concepticondata/conceptlists/PharaoHansen-2020-100.tsv:
```diff
@@ -0,0 +1,101 @@
+PharaoHansen-2020-100-22	22	Vibora	Snake	730	SNAKE
+PharaoHansen-2020-100-23	23	Arbol	Tree	906	TREE
+PharaoHansen-2020-100-24	24	Corteza	Bark	1204	BARK
+PharaoHansen-2020-100-25	25	Hoja	Leef	628	LEAF
```
Same comment as above, is "leef" in the original list?

CarolinHu commented on concepticondata/references/references.bib:
```diff
@@ -3305,6 +3305,27 @@ @Article{Dellert2018
+	booktitle={Lenguas yutoaztecas: historia, estructuras y contacto lingüístico. Homenaje a Karen Dakin},
+	editor={Yañez Rosales, Rosa H.},
+	publisher={Universidad de Guadalajara},
+	address = {Guadelajara},
```
Should the address be Guadalajara as well?

Kristina-Pianykh replied:
Yes, it's a city where the University of Guadalajara is located.

CarolinHu replied:
I know it's a city ;) Sorry, did not make myself clear enough. The university is called GuadAlajara while the address is currently GuadElajara. I think that's a typo (or are there genuinely two ways of writing the name?)

Kristina-Pianykh replied:
You're absolutely right!

LinguList commented:
As to typos: yes, they are in the list. And Guadalajara is the place of the University, so it is the address, yes.

Kristina-Pianykh commented:
@LinguList I just want to draw your attention to PharaoHansen-2020-100-67. In the list in https://github.com/lexibank/pharaocoracholaztecan/blob/master/etc/concepts.tsv the Spanish gloss is 'Come' which I suspect should be 'Venir' (see [the issue](https://github.com/lexibank/pharaocoracholaztecan/issues/10)). It's 'venir' in Miller's list too which was the source for PharaoHansen-2020-100.

LinguList commented:
@Kristina-Pianykh, I think we need to check carefully again and modify the concept list. To explain why, I'll summarize:

1. the concept list reference is the [PDF](https://magnuspharao.files.wordpress.com/2019/06/familiaovecinoscoracholnahuadata.pdf)
2. but our lexibank reference, from which I made the first concept list is [a txt file extracted from the word](https://github.com/lexibank/pharaocoracholaztecan/blob/master/raw/data.txt) which is based on a [word document](https://github.com/lexibank/pharaocoracholaztecan/blob/master/raw/pharaocoracholnahua%20wordlist.docx)
3. it is well possible that the extraction led to problems, or that the official PDF is not in sync with the data.txt.

So we have here again a typical situation in which we need to distinguish the "main source" from the "raw data" source. This means, we should add a column "GLOSS_IN_DATA", and this should be identical with the one I created there, but the SPANISH column would contain the correct version. 

We discussed calling it "alias" or similar, but I think, "GLOSS_IN_DATA", is probably more useful for these cases.

LinguList commented:
ah, sorry, one more thing (I didn't pay enough attention): we should also add Magnus Pharao Hansen to our list of contributors for 2.4, for sharing the concept list with us (as it is not in digital form in the original text).

LinguList commented:
Ah, sorry, then it's a typo. Thanks for spotting!

Kristina-Pianykh commented:
Thank you @LinguList for providig the origial data! I fixed the occasional typos. I'm not sure if there is a need for the "GLOSS_IN_DATA" colum as all the glosses now represet the proper Spanish words. The only thing that leaves doubt is that 6 words are missing the diacritic ' (acute accent) (e.g., the list has _corazon_ instead of _corazón_). If this is important to point out, I'll add an extra column.

LinguList commented:
We need the gloss_in_data column, since we use it to create the lexibank dataset, which lists some English instead of the Spanish words, so this should reflect the concepts exactly as I had them. Accents can then be added in the "real" Spanish.

Kristina-Pianykh commented:
Alright, I see now. So "GLOSS_IN_DATA" will contain the entries as they are rendered in the Lexibank repo (https://github.com/lexibank/pharaocoracholaztecan/blob/master/etc/concepts.tsv) while "SPANISH"- the correctly spelled Spanish glosses with the diacrtics.

LinguList commented:
Yes, @chrzykie will confirm that this was the new strategy we discussed,
but I admit that we did not really communicate this among concepticon
contributors. @xrotwang, @SimonGreenhill, and @chrzykie: we may consider
adding this to the contribution guidelines, for clear documentation with
respect to lexibank? I think it is the safest way to avoid our
re-mapping lists, as we had them in a few lexibank datasets.

LinguList commented on concepticondata/conceptlists.tsv:
```diff
@@ -330,3 +330,5 @@ Baroni-2011-200	Baroni, Marco and Lenci, Alessandro	2011		200	relations	English
 Luangthongkum-2019-341	Luangthongkum, Theraphan	2019		341	questionnaire, areal	English	Karenic varieties spoken in Thailand	https://evols.library.manoa.hawaii.edu/bitstream/10524/52441/D01Luangthongkum2019view.pdf	Luangthongkum2019		The given 341-item list was created in the reconstruction of the phonology and lexicon of Proto-Karen, based on the ten selected Karenic varieties spoken in Thailand. ...
 Pepper-2020-100	Pepper, Steve	2020		100	specific	English	Global		Pepper2020		The items in this list were used for a typological investigation of the morphology and semantics of binominal lexemes, i.e. of concepts typically expressed by nominal compounds, across 106 different languages.	393-394	
+PharaoHansen-2020-100	Pharao Hansen, Magnus	2020		100	basic, areal	Spanish	Southern Uto-Aztecan languages		Pharao2020		The given 100-item list was initially taken from [Miller-1984-100](:ref:Miller-1984-100) and was used in the preliminary analysis of the relationship between the six selected languages within the Southern Uto-Aztecan branch based on a wide range of empirical data.		
+Miller-1984-100	Miller, Wick R.	1984		100	basic, areal	Spanish	Uto-Aztecan languages		Miller1984	Miller1984	The current list represents a modified version of [Swadesh-1955-100](:ref:Swadesh-1955-100) with twelve substitutions (*belly button*, *heavy*, *old*, *rope*, *salt*, *sing*, *sky*, *snake*, *snow*, *vomit*, *wind*, and *year* replace *all*, *full*, *I*, *round*, *say*, *swim*, *that*, *thi ...
```
One question left: "belly button": is this a NEW concpept or an old one? I guess a new one, right? In general, although I appreciate the detail, it may be useful to not list the substitutions, maybe only list one or two as exemples, as one can compare them anyway, and if so, I think it is also good to list the concepticon gloss, so it is clearer, which item we talk about.

AnnikaTjuka replied:
The gloss 'belly button' is usually mapped to 1838 NAVEL. 

I think it's interesting that they did not include 'I' and 'thou' which are very frequently used in other lists and added glosses such as 'snow' instead. @Kristina-Pianykh you can of course also pick out other examples.

chrzyki commented:
`GLOSS_IN_DATA` sounds good to me. See also this associated issue: https://github.com/lexibank/pylexibank/issues/222.

AnnikaTjuka commented:
Thank you @LinguList and @CarolinHu for reviewing this PR! 

@Kristina-Pianykh Please let me know when you are ready to merge, then I'll take another final look at everything.

Kristina-Pianykh commented:
@AnnikaTjuka could you take a final look? I just added Magnus Pharao Hansen as a contributor and updated the description of Miller's list as discussed above.

AnnikaTjuka commented on concepticondata/conceptlists.tsv:
```diff
@@ -330,4 +330,6 @@ Baroni-2011-200	Baroni, Marco and Lenci, Alessandro	2011		200	relations	English
 Luangthongkum-2019-341	Luangthongkum, Theraphan	2019		341	questionnaire, areal	English	Karenic varieties spoken in Thailand	https://evols.library.manoa.hawaii.edu/bitstream/10524/52441/D01Luangthongkum2019view.pdf	Luangthongkum2019		The given 341-item list was created in the reconstruction of the phonology and lexicon of Proto-Karen, based on the ten selected Karenic varieties spoken in Thailand. ...
 Pepper-2020-100	Pepper, Steve	2020		100	specific	English	Global		Pepper2020		The items in this list were used for a typological investigation of the morphology and semantics of binominal lexemes, i.e. of concepts typically expressed by nominal compounds, across 106 different languages.	393-394	
+PharaoHansen-2020-100	Pharao Hansen, Magnus	2020		100	basic, areal	Spanish	Southern Uto-Aztecan languages		Pharao2020		The given 100-item list was initially taken from [Miller-1984-100](:ref:Miller-1984-100) and was used in the preliminary analysis of the relationship between the six selected languages within the Southern Uto-Aztecan branch based on a wide range of empirical data.		
+Miller-1984-100	Miller, Wick R.	1984		100	basic, areal	Spanish	Uto-Aztecan languages		Miller1984	Miller1984	The current list represents a modified version of [Swadesh-1955-100](:ref:Swadesh-1955-100) with twelve substitutions. The pronouns and demonstratives found in Swadesh were omitted and replaced instead with the concepts, denoting simple actions (e.g. *SING*, *VOMIT*) and natural phenomena ( ...
```
I would formulate the second sentence as follows:

The pronouns and demonstratives found in [Swadesh-1955-100](:ref:Swadesh-1955-100) were omitted and replaced with concepts denoting simple actions (e.g., SING, VOMIT) and natural phenomena (e.g., WIND, SNOW, SKY).

AnnikaTjuka commented:
Just a minor remark. Feel free to merge the PR after you edited the sentence. Many thanks!
